### PR TITLE
[Tts] Add new internal API for TTS synthesized PCM

### PR DESCRIPTION
**Pull request by stom-hwang**
### Description of Change ###

Add internal method for playing mode and synthesized pcm event.

### CODE REVIEW

File: src/Tizen.Uix.Tts/Tizen.Uix.Tts/SynthesizedPcmEventArgs.cs [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+using System.ComponentModel;
+
+namespace Tizen.Uix.Tts
+{
+    /// <summary>
+    /// This class holds information related to the TTS SynthesizedPcm event.
+    /// </summary>
+    /// <since_tizen> 11 </since_tizen>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class SynthesizedPcmEventArgs
```
Comment (myroot) [inline src/Tizen.Uix.Tts/Tizen.Uix.Tts/SynthesizedPcmEventArgs.cs:27]: It should inherit from System.EventArgs

Comment (stom-hwang) [reply src/Tizen.Uix.Tts/Tizen.Uix.Tts/SynthesizedPcmEventArgs.cs:27]: Thank you for checking. I think other event args in Tts do not inherit the System.EventAgrs.
So Is it okay to fix it all other classes without ACR?

**Review by urmain — APPROVED** [review]
LGTM~!!

**Review by syeonholic — APPROVED** [review]
It looks good to me.

**Pull request merged**